site/blog/airship-update-march-2020.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Airship Update - March 2020
+author: alex-hughes
+date: 2020-03-11T12:00:00.000Z
+category: Airship 2.0
+---
+
+What follows is the Airship Community Update for the month of March, 2020, brought to you by the Airship Technical
+Committee.<!-- more -->
+
+## **SPOTLIGHT ON ....**
+
+- Daylight Saving Time impact on community meetings. If you do not observe Daylight Saving Time please note that meeting
+times are now one hour earlier than they were previously. For the current meeting list and their schedules check the
+[*Airship Wiki*](https://wiki.openstack.org/wiki/Airship#Get_in_Touch).
+
+<br>
+
+## **AIRSHIP BLOG**
+
+The Airship blog is a great way to keep up with what's going on in the community. The Airship community publishes
+[*blog posts*](https://www.airshipit.org/blog/) regularly, including the recent Airship 2.0 Blog series. These blog
+posts introduce the changes from Airship 1.0 to Airship 2.0, highlight new features, and detail the evolution of each
+component. The first six Airship 2.0 Blog posts are already available:
+
+- [*Airship Blog Series 1 - Evolution Towards 2.0*](
+  https://www.airshipit.org/blog/airship-blog-series-1-evolution-towards-2.0.html)
+- [*Airship Blog Series 2 - An Educated Evolution*](
+   https://www.airshipit.org/blog/airship-blog-series-2-an-educated-evolution.html)
+- [*Airship Blog Series 3 - Airship 2.0 Architecture High Level*](
+   https://www.airshipit.org/blog/airship-blog-series-3-airship-2.0-architecture-high-level.html)
+- [*Airship Blog Series 4 - Shipyard - an Evolution of the Front Door*](
+   https://www.airshipit.org/blog/airship-blog-series-4-shipyard-an-evolution-of-the-front-door.html)
+- [*Airship Blog Series 5 - Drydock and Its Relationship to Cluster API*](
+  https://www.airshipit.org/blog/airship-blog-series-5-drydock-and-its-relationship-to-cluster-api.html)
+- [*Airship Blog Series 6 - Armada Growing Pains*](
+   https://www.airshipit.org/blog/airship-blog-series-6-armada-growing-pains.html)
```

Comment:
Seems like the order on these should be inverted with the latest posts first

Reply:
done.